application/models/post_fixtures_test.go

```diff
@@ -196,16 +196,25 @@ func CreateFixtures_Posts_FindByUser(ms *ModelSuite) PostFixtures {
 	}
 	createFixture(ms, &trusts)
 
-	users := createUserFixtures(ms.DB, 2).Users
+	users := createUserFixtures(ms.DB, 4).Users
 
-	// both users are in org 0, but need user 0 to also be in org 1
 	createFixture(ms, &UserOrganization{
 		OrganizationID: orgs[1].ID,
 		UserID:         users[0].ID,
 		AuthID:         users[0].Email,
 		AuthEmail:      users[0].Email,
 	})
 
+	uo, err := users[2].FindUserOrganization(orgs[0])
+	ms.NoError(err)
+	uo.OrganizationID = orgs[2].ID
+	ms.NoError(DB.UpdateColumns(&uo, "organization_id"))
+
+	uo, err = users[3].FindUserOrganization(orgs[0])
+	ms.NoError(err)
+	uo.OrganizationID = orgs[1].ID
+	ms.NoError(DB.UpdateColumns(&uo, "organization_id"))
+
 	posts := createPostFixtures(ms.DB, 8, 0, false)
 	posts[1].OrganizationID = orgs[1].ID
 	posts[2].Status = PostStatusCommitted
```

Comment:
Maybe choose a status that's going to still be in the code, e.g. `PostStatusAccepted`?

Reply:
Can't go from OPEN (as created by createPostFixtures) to ACCEPTED in one step, so this code would have to be edited again to remove the double-update when dev2 is merged. Hopefully this won't be a merge conflict since line 220 is separated from the new lines.